### ISSUE TITLE
Fix Glowing Bear showing up twice as recipe

### ISSUE THREAD

**Issue by mhatvan** [getferdi/recipes#612]
**Describe the bug**
The recipe only exists once in the repo, but shows up twice on the services screen.

**Comment by vraravam** [comment]
This is due to [this line](https://github.com/getferdi/ferdi/blob/ee7d9b0af920cc2021028faa52efe78d6ce0880a/src/internal-server/app/Controllers/Http/RecipeController.js#L65) - the code connects to the remote server and hits [this api](https://api.getferdi.com/v1/recipes/search?needle=bear) - which returns
```json
0:  
  author: "Jonathan Juursema <dev@jonathanj.nl>"
  featured: false
  id: "glowingbear"
  name: "Glowing Bear"
  version: "1.0.0"
  icons:  
  png: "https://cdn.franzinfra.com/recipes/dist/glowingbear/src/icon.png"
  svg: "https://cdn.franzinfra.com/recipes/dist/glowingbear/src/icon.svg"
```
As can be seen (presence of the 'author', 'icons.png') this is the OLD version of the recipes JSON representation. Also as seen, the `id` of this recipe hit is `glowingbear` - which is different from the one that is packaged inside Ferdi. Thus, we get 2 hits.

1. The temporary fix is to redeploy the server so that the latest and greatest versions of the recipes are served.
2. The other temporary fix could be to turn off the `CONNECT_WITH_FRANZ` - but, that will not work for those users who are already using ferdi.
3. The long-term idea is to completely remove the packaged recipes and only retrieve them from the server.

On a minor note, I will refactor the code such that the server that it hits need not always be the hardcoded remote - instead, it can retrieve from the internal-server for those who do not use a login

**Comment by vraravam** [comment]
@kytwb - how can we deploy the new server? Will it pick up a new version of the packaged recipes?

**Comment by vraravam** [comment]
Update: I have now confirmed that the local server itself is not adding this data. It is, in turn, [calling the ferdi server](https://github.com/getferdi/ferdi/blob/408ca48337fea6269bc6026f9fee35bf0c5df9c6/src/internal-server/app/Controllers/Http/RecipeController.js#L67) and sending the results back. So, unless we find a way for the server to decouple it from the franz api [over here](https://github.com/getferdi/server/blob/6556717654055d4cc0c3f8e1a9e45686c4ed6ca6/app/Controllers/Http/RecipeController.js#L149) - we will continue to get these duplicate/erroneous results.

imo, we have moved sufficiently away from Franz that we should remove all runtime ties to it atm.

cc @kytwb @vantezzen @kris7t

**Comment by mhatvan** [comment]
Thanks Vijay for the investigation!

Would also vote in favor of decoupling it.

**Comment by kytwb** [comment]
Fixed via getferdi/server@e4a7fcb.

![Screen Shot 2021-12-31 at 2 55 01 PM](https://user-images.githubusercontent.com/412895/147826841-d89cf699-43a3-42de-8481-251b443cedbe.png)
![Screen Shot 2021-12-31 at 2 55 21 PM](https://user-images.githubusercontent.com/412895/147826860-f6635e0e-5f6c-4bf4-a814-35fe645cf1e8.png)
![Screen Shot 2021-12-31 at 2 55 31 PM](https://user-images.githubusercontent.com/412895/147826867-ba067b6f-4cad-4836-a1f9-5abdae87a78b.png)
